pkgs/games/marble-marcher/default.nix

```
@@ -0,0 +1,67 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, cmake, eigen, sfml, libglvnd }:
+let
+  pname = "marble-marcher";
+
+  # The game doesn’t have any versions in GitHub.
+  # Just taking a commit from the top of the “master” branch.
+  version = "unstable-2019-04-22";
+
+  src = fetchFromGitHub {
+    owner = "HackerPoet";
+    repo = "MarbleMarcher";
+    rev = "478afb8ea9c894d6d8b961583aa557e2f63d6198";
+    sha256 = "1kpn9wnf14wm7867dp3lhb10065h4vzy101gab18zkphlwl1dbwq";
+  };
+  assets = stdenv.mkDerivation {
+    pname = "${pname}-assets-pack";
+    inherit version src;
+    installPhase = ''
+      mkdir -p -- "$out/assets" || exit
+      cp -- "$src/assets/"* "$out/assets" || exit
```

```suggestion
      mkdir -p "$out/assets"
      cp "$src/assets/"* "$out/assets"
```

We control $out and it can#t contain -- also if the commands fail nix will stop anyway.